mobile/library/swift/cxx/LogLevel+Cxx.swift

```diff
@@ -0,0 +1,30 @@
+@_implementationOnly import EnvoyCxxSwiftInterop
+
+extension LogLevel {
+  func toCXX() -> Envoy.Platform.LogLevel {
+    switch self {
+    case .trace:
+      return .trace
+    case .debug:
+      return .debug
+    case .info:
+      return .info
+    case .warn:
+      return .warn
+    case .error:
+      return .error
+    case .critical:
+      return .critical
+    case .off:
+      return .off
+    }
+  }
+}
+
+// MARK: - Test Helpers
+
+extension LogLevel {
+  func toCXXDescription() -> String {
```

Comment:
this name is a bit misleading - seems to suggest that it returns a c++ type. Any ideas as to whether we could make it clearer that this does not return c++ type?

Reply:
I'll rename to `cxxDescription: String`, which is hopefully clearer that it's a description of a C++ type, rather than 
returning a C++ string.